src/components/CarouselContainer.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+import { cssScale } from "src/common/constants";
+
+interface ICarouselContainer {
+  width?: string,
+  height?: string
+}
+
+const CarouselContainer = styled.div<ICarouselContainer>`
```

Comment:
I don't think that this would be re-used. Let's move it to the file where it's used.